app/views/kitchens/show.html.erb

```diff
@@ -31,20 +31,19 @@
           </div>
         </div>
       </div>
-
-      <%= link_to "Back to Kitchens", kitchens_path, class: "btn btn-flat" %>
-
       <div class="kitchen-show-card">
         <div class="flex-align">
-          <h6 class="no-mb">Share</h6>
+          <h6 class="no-mb">Share this kitchen</h6>
 
           <div class="flex">
             <i class="fa-brands fa-facebook-square me-2"></i>
             <i class="fa-brands fa-twitter-square me-2"></i>
             <i class="fa-brands fa-pinterest-square me-2"></i>
           </div>
-
         </div>
+        <div class="kitchen-card-actions">
```

Comment:
Can you remove this "back to kitchens path" because in a minutes its going to be in the navbar (like HOME) so we dont do that anymore